setup.py

```diff
@@ -26,6 +26,7 @@
         "microcosm-caching>=0.2.0",
         "microcosm-daemon>=1.0.0",
         "microcosm-logging>=1.3.0",
+        "isort<5.0.0",
```

Comment:
Is this required for build to pass? Ideally we wouldn't have such a restriction in a library, as it makes it more complicated to to upgrade isort in the future for the services using this library.

Reply:
Unfortunately. 
`Lint` job fails because of that.
We have this issue in all repositories at this moment.

Reply:
Just saw Tony's comment in Slack on that. let me try to kick the build again and see what happens

Reply:
Looks like it's not required anymore!